doc/apiv3-documentation.apib

```diff
@@ -2271,6 +2271,397 @@ Lists the columns available for work packages of this project in the form of fie
+                                {
+                                  "href": "/api/v3/relations/1",
+                                  "method": "PATCH"
+                                },
+                                "delete":
+                                {
+                                  "href": "/api/v3/relations/1",
+                                  "method": "DELETE"
+                                },
+                                "from":
+                                {
+                                    "href": "/api/v3/work_packages/42",
+                                    "title": "Steel Delivery"
+                                },
+                                "to":
+                                {
+                                    "href": "/api/v3/work_packages/84",
+                                    "title": "Bending the steel"
+                                }
+                            },
+                            "_type": "Relation",
+                            "id": 1,
+                            "type": "precedes",
+                            "reverseType": "follows",
+                            "description": "We can't bend the steel before it's been delivered!",
+                            "delay": 0
+                        }
+                    ]
+                }
+            }
+
+## List Relations [GET]
+
+Lists all relations according to the given (optional, logically conjunctive) filters and ordered by ID.
+The response only includes relations between work packages which the user is allowed to see.
+
++ Parameters
+    + offset = `0` (optional, integer, `5`) ... Page number inside the requested collection
+
+    + pageSize (optional, integer, `20`) ... Number of elements to display per page
+
+    + from (optional, integer, `42`) ... ID of work package from which the filtered relations emanates
+
+    + to (optional, integer, `84`) ... ID of work package to which this related points
+
+    + involved (optional, integer, `24`) ... ID of either the `from` or the `to` work package.
+
+    + type (optional, string, `blocks`) ... The type of relation to filter by
+
++ Response 200 (application/hal+json)
+
+    [Relations][]
+
+## Create Relation [POST]
+
+When calling this endpoint the client provides a single object, containing at least the properties and links that are required, in the body.
+The required fields of a Relation can be found in its schema, which is embedded in the respective form.
+Note that it is only allowed to provide properties or links supporting the write operation.
+
++ Request Create Relation (application/json)
+
+    + Body
+
+        {
+            "_links":
+            {
+              "from": "/api/v3/work_packages/42",
+              "to": "/api/v3/work_packages/84"
+            }
+            "type": "duplicates",
+            "description": "This is the same thing. Let's track it in one place only, shall we?"
+        }
```

Comment:
aglio complains about this block:

`message-body asset is expected to be a pre-formatted code block, every of its line indented by exactly 12 spaces or 3 tabs (warning code 10)`